### ISSUE TITLE
New version of Font Awesome downloaded - how do I update sketch-iconfont ?

### ISSUE THREAD

**Issue by steveharman** [keremciu/sketch-iconfont#162]
Something I've never understood and couldn't find in the documentation.  Font Awesome (FA) regularly update their icon font to include new fonts.

What do we do to install the updated FA fonts into sketch-iconfont?  

The FA download includes an "svgs" folder that contains hundreds of .svg files in several weights - we don't have to instal each one manually, do we?

Thanks,

**Comment by keremciu** [comment]
@steveharman good point! I wrote in an issue how to handle this case but maybe I need to document it for further usage.

Let me look at new updates on their icon-font files

**Comment by steveharman** [comment]
Many thanks.  Am happy to test out any docs / instructions @keremciu

**Comment by steveharman** [comment]
HI,
Has there been any news with this?  

It must be one of the most common things people need to do.  "New Font Awesome release > update to latest font in icon-font plugin".  How?

Thanks

**Comment by humphreyja** [comment]
I've found that it is easier to just add the font to your system.  Then you can simply just use that font in sketch.  It uses ligatures to convert `leaf` into the appropriate icon.  Doesn't come with the search ability but should work with any upgrades to font awesome
